android-components/components/ui/widgets/src/main/java/mozilla/components/ui/widgets/Extentions.kt

```diff
@@ -0,0 +1,28 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.ui.widgets
```

Comment:
wasn't sure where to put it, is there a better place?

Reply:
This seems like a good place!